### ISSUE TITLE
Maintain scroll position when content changes

### ISSUE THREAD

**Issue by nesium** [prose-im/prose-app-web#77]
When you've typed enough text into the message textfield that it contains multiple lines and scrolled the conversation to the bottom (so that you can see the last message), typing another character will move the last message behind the textfield.

https://github.com/prose-im/prose-app-web/assets/39174/9c0e18a0-3305-4513-996f-179c43f43398

**Comment by valeriansaliou** [comment]
Is it still happening for you? I can't seem to reproduce, at least on `master` latest.

**Comment by nesium** [comment]
Seems to work fine now.

**Comment by nesium** [comment]
I'm going to reopen this as an issue to maintain the scroll position. Two new scenarios that I've discovered:

1. When drafting a multi-line message the textfield grows above the (last) message I'm referring to, even if the message list is scrolled to the bottom.
2. When someone adds a reaction to a message above the last message, the last message is pushed down behind the textfield.
3. When someone adds a reaction to the last message, the reaction is partially truncated.

In summary I would define the rules as: 
- When the message list is scrolled to the bottom, it should stay at the bottom.
- Content in the visible area should never be pushed down by changes above the visible area (i.e. reactions added or messages being updated)
